src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -740,7 +740,7 @@ internal enum ErrorCode
         WRN_BadXMLRef = 1574,
         ERR_BadStackAllocExpr = 1575,
         ERR_InvalidLineNumber = 1576,
-        //ERR_ALinkFailed = 1577,               No alink usage in Roslyn
+        ERR_TypeForwardedToMultipleAssemblies = 1577,
```

Comment:
I don't think we should reuse this error code. It looks like native compiler was using for all possible ALink errors.